.github/workflows/ci.yml

```
@@ -75,3 +75,110 @@ jobs:
+          aws-region: ${{ vars.AWS_REGION }}
+          role-session-name: ${{ github.run_id }}
+
+      - name: Login to Amazon ECR
+        id: ecr
+        uses: aws-actions/amazon-ecr-login@v2.1.6
+
+      - name: Build and push backend image
+        id: image
+        env:
+          ECR_REGISTRY: ${{ steps.ecr.outputs.registry }}
+          ECR_REPOSITORY: ${{ vars.ECR.REPOSITORY }}
```

actionlint (1.7.12)

[error] receiver of object dereference "repository" must be type of object but got "string"

(expression)